src/main/kotlin/org/jetbrains/kotlin/jupyter/config.kt

```diff
@@ -1,9 +1,23 @@
 package org.jetbrains.kotlin.jupyter
 
+import com.beust.klaxon.JsonObject
+import com.beust.klaxon.Parser
+import org.json.JSONObject
 import org.slf4j.LoggerFactory
 import java.io.File
+import java.io.StringReader
+import java.nio.file.Paths
 import kotlin.script.experimental.dependencies.RepositoryCoordinates
 
+val LibrariesDir = "libraries"
+
+val LocalSettingsPath = Paths.get(System.getProperty("user.home"), ".jupyter_kotlin").toString()
+
+val GitHubApiHost = "api.github.com"
+val GitHubRepoOwner = "kotlin"
+val GitHubRepoName = "kotlin-jupyter"
+val GitHubApiPrefix = "https://$GitHubApiHost/repos/$GitHubRepoOwner/$GitHubRepoName/contents/"
```

Comment:
What about adding branch here? It would be helpful when changing this code and testing new functionality. I'd even made it a system property.